build/WinObjC.Frameworks/WinObjC.Frameworks.nuproj

```diff
@@ -110,6 +110,9 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\UIKit.Xaml\dll\UIKit.Xaml.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\UIKit\dll\UIKit.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\WebKit\dll\WebKit.vcxproj" />
+
+  <!-- NuProj Project References. Ensure that these only get evaluated once by checking that this isn't the second time through this project -->
+  <ItemGroup Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
```

Comment:
><ItemGroup Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'"> [](start = 2, length = 78)

So this check is needed anytime a nuproj has a ProjectReference on another nuproj? As in, we will need to make sure to edit the .nuproj file and add this check whenever we add a reference to another nuproj?
Additionally, when does SkipGetPackageContentsForOtherPlatforms get set?